src/views/components/student-panel.jsx

```diff
@@ -23,30 +23,57 @@ export default class StudentPanel extends ListenerComponent {
     }
 
     renderEquipment() {
-        if(this.props.student.items.length === 0) {
-            return <i className='equipment-none'>Student has no equipment checked out.</i>;
+        if(this.props.student.items.length === 0 && this.props.student.models.length === 0) {
+            return (<i className='equipment-none'>Student has no equipment checked out.</i>);
         }
+
+        let modelCounts = [];
+        this.props.student.models.forEach(model => {
```

Comment:
This logic shouldn't exist in the render function?
There has got to be a cleaner way to identify the amount of duplicate models a student has checked out.